README.rst

```diff
@@ -132,7 +132,7 @@ First You need to create a `config-file <https://privacyidea.readthedocs
 
 Then create the database and encryption key::
```

Comment:
Then we could also change this to 

"Then create the database tables and the encryption key"

Reply:
Done in: a5b50932686161a11509b56b4a98ac3665d0d493